module/Makefile.in

```diff
@@ -115,7 +115,8 @@ modules_uninstall: modules_uninstall-@ac_system@
 
 cppcheck-Linux:
 	@CPPCHECK@ -j@CPU_COUNT@ --std=c99 --quiet --force --error-exitcode=2 \
-		--inline-suppr --suppress=noValidConfiguration \
+		--inline-suppr --suppress=unmatchedSuppression \
+		--suppress=noValidConfiguration \
```

Comment:
[nit] it would be easier to read changes to this in the future if we formatted it with one flag per line

Reply:
Corrected in https://github.com/openzfs/zfs/commit/b9826acd531351530e163e8da5768cb1b61fb8e7